src/app/template-library/template-library.component.scss

```diff
@@ -0,0 +1,85 @@
+.category-card-container {
+  height: 220px;
+  width: 100%;
+  border-radius: 7px;
+  position: relative
+}
+
+.category-card-container-small {
+  height: 120px;
+  width: 100%;
+  border-radius: 7px;
+  position: relative
+}
+
+.category-card-background {
+  height: 100%;
+  width: 100%;
+  border-radius: 7px;
+  position: absolute;
+  background-image:url('https://images.unsplash.com/photo-1581089778245-3ce67677f718?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+.category-card-overlay {
+  height: 100%;
+  width: 100%;
+  background: rgba(0, 64, 229, 0.6);
+  border-radius: 7px;
+  position: absolute;
+}
+
+.category-card-title-container {
+  height: 100%;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.category-heading {
+  margin-top: 50px;
+  color: #727272;
+}
+
+@media (min-width:1300px) {
+  .template-library-container {
+    width: 80%;
+    padding-top: 80px;
```

Comment:
Is there any reason that we padding top that much. It looks wired to have that many spaces on top.
Also, the left sidebar looks not good on a screen width smaller than 1300px. Maybe add some margin-left

Reply:
I have it in to give space to the top bar once it is embedded in the frame (80px is probably too much though). Do you think I should remove it completely and let the core team adjust the padding of the frame in the admin?

Reply:
Oh, if it is for the top bar, don't worry about that, the top bar won't be part of this page. The iframe will be put under the top bar, so don't need to leave spaces for it.